src/packages/media/media/workspace/views/collection/media-workspace-view-collection.element.ts

```diff
@@ -0,0 +1,83 @@
+import type {
+	UmbCollectionBulkActionPermissions,
+	UmbCollectionConfiguration,
+} from '../../../../../core/collection/types.js';
```

Comment:
Looks like you import from another module here. Please consider if an export is missing.